templates/test/dev/cluster-template-custom-builds-load.yaml

```diff
@@ -104,21 +108,110 @@ spec:
       owner: root:root
       path: /etc/kubernetes/azure.json
       permissions: "0644"
+    - content: |
+        #!/bin/bash
+
+        set -o nounset
+        set -o pipefail
+        set -o errexit
+        [[ $(id -u) != 0 ]] && SUDO="sudo" || SUDO=""
```

Comment:
I don't see `sudo` being called in this script, so this line could be removed.

Reply:
Ditto to above